h2o-r/ensemble/h2oEnsemble-package/R/performance_cv.R

```diff
@@ -0,0 +1,80 @@
+h2o.ensemble_performance_cv <- function(object, training_frame=train, score_base_models=TRUE){
+  out <- vector("list", length(object))
+  for (i in 1:length(out)){
+    out[[i]] <- h2o.ensemble_performance(object[[i]], newdata=training_frame[-object[[i]]$tt_ind,], score_base_models=score_base_models)
+  }
+  names(out) <- names(object)
+  class(out) <- "h2o.ensemble_cv_performance"
+  return(out)
+}
+
+
+### print function for class 'h2o.ensemble_cv_performance'
+print.h2o.ensemble_cv_performance <- function(x, metric = c("AUTO", "logloss", "MSE", "AUC", "r2"), ...) {
+  
+  # We limit metrics to those common among all possible base algos
+  metric <- match.arg(metric)
+  if (metric == "AUTO") {
+    if (class(x[[1]]$ensemble) == "H2OBinomialMetrics") {
```

Comment:
we use logloss for classification and deviance for regression, pretty much everywhere else.